src/masternodes/rpc_proposals.cpp

```diff
@@ -3,8 +3,27 @@
 
 #include <functional>
 
-UniValue propToJSON(CPropId const& propId, CPropObject const& prop)
+UniValue propToJSON(CPropId const& propId, CPropObject const& prop, CCustomCSView const& view)
 {
+    CDataStructureV0 cfpMajority{AttributeTypes::Governance, GovernanceIDs::Proposals, GovernanceKeys::CFPMajority};
+    CDataStructureV0 vocMajority{AttributeTypes::Governance, GovernanceIDs::Proposals, GovernanceKeys::VOCMajority};
+    CDataStructureV0 quorumKey{AttributeTypes::Governance, GovernanceIDs::Proposals, GovernanceKeys::Quorum};
+
+    auto attributes = view.GetAttributes();
+    if (!attributes)
+        throw JSONRPCError(RPC_INTERNAL_ERROR, "Attributes access failure");
+    auto quorum = attributes->GetValue(quorumKey, Params().GetConsensus().props.quorum); 
+
+    uint32_t approvalThreshold{0};
+    switch(prop.type) {
+        case CPropType::CommunityFundProposal:
+            approvalThreshold = attributes->GetValue(cfpMajority, Params().GetConsensus().props.cfp.approvalThreshold) / 10000;
+            break;
+        case CPropType::VoteOfConfidence:
+            approvalThreshold = attributes->GetValue(vocMajority, Params().GetConsensus().props.voc.approvalThreshold) / 10000;
+            break;
+    }
+
```

Comment:
Can use `GetMajorityFromAttributes` and `GetQuorumFromAttributes` (which handles `vocEmergencyQuorum` as well)

Reply:
Should rename `GetMajorityFromAttributes` to `GetApprovalThresholdFromAttributes` at the same time